imap/ctl_deliver.c

```diff
@@ -75,7 +76,21 @@ int main(int argc, char *argv[])
     char *days = NULL;
     enum { DUMP, PRUNE, NONE } op = NONE;
 
-    while ((opt = getopt(argc, argv, "C:drE:f:")) != EOF) {
+    /* keep this in alphabetical order */
+    static const char *const short_options = "C:E:df:";
```

Comment:
Where did the r option go?

Reply:
The commit message for this change explains:

>     ctl_deliver: remove dead '-r' option from getopt string
>     
>     Implementation was removed by 8d6cf338198bae06a7bc7ebb78ba1e5544029e4e

Looking at that commit's date, the "r" in the getopt string had been dangling around with no implementation for nearly twenty years... and it was already commented as "deprecated" at the time it was removed!

So I guess it was waiting all this time for someone to look at it really closely, and I finally did. :)

Reply:
Sorry for that. I read the commit message but the only special-handled option that sticked was the C option.